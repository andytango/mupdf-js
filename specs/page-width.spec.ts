import {
  forExamplePages,
  loadExampleFile,
  TEST_DPI,
} from "./helpers/example-file";

describe("pageWidth", () => {
  forExamplePages("should return the page width", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.pageWidth(doc, page, TEST_DPI)).toMatchSnapshot();
  });
});
