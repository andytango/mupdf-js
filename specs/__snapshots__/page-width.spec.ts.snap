// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pageWidth Page [1 / 4] - should return the page width 1`] = `612`;

exports[`pageWidth Page [2 / 4] - should return the page width 1`] = `612`;

exports[`pageWidth Page [3 / 4] - should return the page width 1`] = `612`;

exports[`pageWidth Page [4 / 4] - should return the page width 1`] = `612`;
